setup.py

```diff
@@ -2,6 +2,11 @@
 import os
 from setuptools import setup, find_packages
 
+if not os.listdir('./asdf-standard'):
```

Comment:
I'm not sure if this script is always executed with asdf as the working directory, perhaps we should be defensive here and list the directory relative to this file's location:

```python
if not any((Path(_file_).parent / "asdf-standard").iterdir()):
  # ...
```

Reply:
See 7bf4fd3.